pkg/plugins/runtime/gateway/gateway_route_generator.go

```diff
@@ -192,6 +192,19 @@ func makeHttpRouteEntry(name string, rule *mesh_proto.MeshGatewayRoute_HttpRoute
 
 			entry.RequestHeaders.Delete = append(
 				entry.RequestHeaders.Delete, h.GetRemove()...)
+		} else if h := f.GetRewrite(); h != nil {
```

Comment:
super nitpick: it seems like the variable name is the first letter of the filter:
* r := f.GetRedirect()
* m := f.GetMirror()
* h := f.GetRequestHeader()

so it should be:
* r := f.GetRewrite()